pkg/cmd/helm/build/build.go

```diff
@@ -119,7 +133,7 @@ func (o *Options) Run() error {
 		}
 
 		chartDef := &chart.Chart{}
-		if exists {
+		if exists && !o.OCI {
```

Comment:
This looks wrong to me. Shouldn't you check if dependency.Repository starts with oci:// and then skip it?

Reply:
Yes, that's probably the way